packages/browser/examples/rails/README.md

```diff
@@ -45,9 +45,9 @@ yarn add @airbrake/browser
 In your main application pack, import `@airbrake/browser` and configure the client.
 
 ```js
-import Airbrake from '@airbrake/browser';
+import Notifier from '@airbrake/browser';
```

Comment:
Should be either
- `import Airbrake` and new Airbrake.Notifier
- `import {Notifier}` and new Notifier